src/components/seek_time/seek_time.js

```diff
@@ -89,8 +90,17 @@ export default class SeekTime extends UIObject {
       this.$el.css('left', "-100%")
     }
     else {
-      var seekTime = this.hoverPosition * this.duration
-      var currentSeekTime = formatTime(seekTime)
+      if (this.actualLiveTime) {
+        var d = new Date(), e = new Date(d);
+        var secondsSinceMidnight = (e - d.setHours(0,0,0,0)) / 1000;
+        var seekTime = (secondsSinceMidnight - this.duration) + (this.hoverPosition * this.duration)
+        if (seekTime < 0) {
+          seekTime = (3600 * 24) + seekTime
+        }
+      } else {
+        var seekTime = this.hoverPosition * this.duration
+      }
```

Comment:
I came to think of a possible more flexible option. How about allowing a date object as an option where we add seek position/duration to that date. This way we can set server side time and let users go around the problem with wrong timezones and other client side problems.